lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -612,7 +612,7 @@ sub allocate_remember_me_token {
         my $token = generate_token();
 
         my $key = "$normalized_name|$token";
-        $self->redis->add($key, 1);
+        $self->redis->set($key, 1);
```

Comment:
The difference between `add` ([SETNX](http://redis.io/commands/setnx)) and `set` was that `add` only set the key if it didn’t exist before. However, as the value is always 1 anyway, the only difference would be that setting a value also resets the expire time; but we explicitly re-set it in the next command. So, no added value in having a separate `add`.